client/src/utils/ajax.ts

```diff
@@ -251,3 +259,8 @@ export function putUserUpdateEmail(email: string): Promise<void> {
 export function putVerifyCert(certSlug: string): Promise<void> {
   return put('/certificate/verify', { certSlug });
 }
+
+/** DELETE **/
+export function deleteWebhookToken(): Promise<void> {
+  return deleteRequest('/user/webhook-token', {});
```

Comment:
```suggestion
  return deleteRequest('/user/webhook-token');
```
Again, seem unnecessary.